brian2tools/mdexport/mdexporter.py

```diff
@@ -102,19 +106,26 @@ def build(self, direct_call=True, debug=False, expander=None,
             
             # Check if Pandoc is installed
             try:
-                subprocess.check_call(["pandoc"])
+                subprocess.check_call(["pandoc", "--version"], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL)
                 formats_extensions = {'latex':'.tex', 'html':'.html', 'pdf':'.pdf'}
-                if format == "all":
-                    formats = ['latex', 'html', 'pdf']
+                if type(Additionalformat) == str:
```

Comment:
Normally, it is recommended to use `isinstance` instead of `type`, see e.g. https://docs.astral.sh/ruff/rules/type-comparison/